src/kboot.c

```diff
@@ -2436,6 +2483,8 @@ int kboot_prepare_dt(void *fdt)
         return -1;
     if (dt_set_multitouch())
         return -1;
+    if (dt_set_sep())
```

Comment:
This is going to blow up on DTs without the SEP alias. That needs to be handled gracefully for backwards compat.